### PR TITLE
Package test on CI

### PR DESCRIPTION
**Pull request by IAlibay**
Included in this PR:
  - package build & test on CI
  - daily cron conda install & test

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -8,7 +8,7 @@ on:
       - main
   schedule:
     # At 07:00 UTC on Monday and Thursday.
-    - cron: "0 7 * * 1,4"
+    - cron: "0 7 * * *"
```
Comment (IAlibay) [inline .github/workflows/ci.yaml:11]: As per today's discussion - run cron nightly now

File: .github/workflows/conda_cron.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+concurrency:
+  group: "${{ github.workflow }}-${{ github.ref }}"
+  cancel-in-progress: true
+
+defaults:
+  run:
+    shell: bash -l {0}
+
+jobs:
+  condacheck:
```
Comment (IAlibay) [inline .github/workflows/conda_cron.yaml:16]: This should hopefully meet everything on your wishlist @richardjgowers - including a rough "raise an issue if it fails" (which we can iterate though / refine at a later point if folks want dashboards, etc...).

**Review by dwhswenson — APPROVED** [review]
LGTM for now (caught one unimportant typo).

I think we may iterate on the issue raising script in the future (e.g, would love to have it link to the failed run, if we can figure that out) but this will already be useful as is.

**Comment by IAlibay** [comment]
> LGTM for now (caught one unimportant typo).
> 
> I think we may iterate on the issue raising script in the future (e.g, would love to have it link to the failed run, if we can figure that out) but this will already be useful as is.

So linking to the fail run is doable (we do so here: https://github.com/MDAnalysis/mdanalysis/blob/develop/maintainer/ci/darker-outcomes.py#L64), I just didn't want to make this complicated until we're really bought into the solution.

**Pull request merged**